src/tasks/barfTasks.ts

```diff
@@ -125,34 +137,15 @@ const steveAdventures: Map<Location, number[]> = new Map([
   [$location`The Haunted Laboratory`, [1, 1, 3, 1, 1]],
 ]);
 
-enum Sobriety {
-  SOBER = "sober",
-  DRUNK = "drunk",
-  EITHER = "either",
-}
+let digitize = -1;
 
-/**
- * Describes an action we could take as part of our barf-turn loop.
- * Has a name, a a function to determine availability, and a function that executes the turn.
- * Execute function returns whether we succeeded.
- */
-type AdventureAction = {
-  name: string;
-  available: () => boolean;
-  execute: () => boolean;
-  spendsTurn: boolean | (() => boolean);
-  sobriety: Sobriety;
-};
+const needTurns: () => boolean = () => myAdventures() === 1 + globalOptions.saveTurns && sober();
```

Comment:
I think this is pretty ugly with the type description; I'd just make it 
```suggestion
const needTurns = () => myAdventures() === 1 + globalOptions.saveTurns && sober();
```